src/main/scala/is/hail/rvd/RVD.scala

```diff
@@ -43,7 +43,7 @@ object RVDSpec {
       val f = path + "/parts/" + p
       val in = hConf.unsafeReader(f)
       HailContext.readRowsPartition(rowType, codecSpec)(0, in)
-        .map(rv => new UnsafeRow(rowType, rv.region.copy(), rv.offset))
+        .map(rv => new UnsafeRow(rowType, rv.region.copy(), rv.offset).toSafeRow)
```

Comment:
Rather than toSafeRow, can we have `SafeRow(rowType, region, offset)`?  I think that's shorter (at the use) and expressions the intention more clearly.